### DISCUSSION TITLE
일부 제목이 다크 모드에서 보이지 않고, 목차에 포함되지 않는 현상

### DISCUSSION THREAD

**Discussion by limehee** [pronist/hELLO#261] [Q&A]
'제목 1'로 설정된 내용

'2. Application Class Data Sharing(AppCDS)'
'3. Spring AOT (Ahead-of-Time)'

이 다크 모드에서 마우스로 블럭 처리하지 않는 한 보이지 않고, 목차에서 누락됩니다.

해당 내용을 제목 1, 2, 3으로 설정한 경우 동일하게 표시가 되지 않고 있으며, 본문으로 설정할 경우 정상적으로 보입니다.

블로그 링크: https://limehee.tistory.com/3

**Comment by pronist** [comment]
@limehee 테스트 해본 결과, 해당 문제의 트리거를 파악할 수 없습니다. 

문제가 발생할 수 있는 가능성

1. HTML 문서 구조가 깨짐
글쓰기 - HTML 모드에서 HTML 문서 구조가 올바르게 구성 되어있지 않을 가능성이 있습니다.

2. 복사 & 붙여넣기
1번과 같은 맥락으로써, 다른 플랫폼에서 글을 복사 & 붙여넣기로 글을 작성한 경우 티스토리 에디터에서 임의로 이를 해석하기 때문에
티스토리의 기본 문서구조와 호환되지 않아 문제가 발생할 가능성이 있습니다.

시도해볼만한 해결 방법으로는 다음과 같습니다.

1. HTML 모드에서 태그의 열고 닫는 부분이 정상적으로 매칭되어 있는지 체크
2. 처음부터 끝까지 티스토리 기본 에디터에서 글 재작성
3. 제목에 `4. Docker 빌드 프로세스에 적용하기` 와 같은 의미를 알 수 없는 아이콘 제거

문제의 원인이 파악이 되지 않기 때문에 근본적인 해결 방법 제시나 패치는 현재 어려운 점 양해 바랍니다.

**Reply by limehee** [reply]
해당 부분 확인하고 조치해보겠습니다. 감사합니다 :)
